agent/handlers/v2_server_setup.go

```diff
@@ -21,78 +21,87 @@ import (
 	"github.com/aws/amazon-ecs-agent/agent/config"
 	"github.com/aws/amazon-ecs-agent/agent/credentials"
 	"github.com/aws/amazon-ecs-agent/agent/engine/dockerstate"
+	handlersutils "github.com/aws/amazon-ecs-agent/agent/handlers/utils"
```

Comment:
The name of the file is confusing, as both v3 and v2 server are set in this file, can you restructure this to the following:
```
-handlers
-----server_setup.go(call v1/v2/v3 server_setup to start all servers)
-----v1
---------v1_server_setup.go
-----v2
----------v2_server_setup.go
-----v3
----------v3_server_setup.go
```